File: python/importacion_de_modulos/crear_modulo_de_validaciones/validaciones.py
def validar_nombre(nombre):
    if not nombre.strip():
        return False, "El nombre no puede estar vacío"
    if not nombre.isalpha():
        return False, "El nombre solo debe contener letras"
    return True, "Nombre válido"
def validar_telefono(telefono):
    if not telefono.isdigit():
        return False, "El teléfono solo debe contener números"
    if len(telefono) < 7:
        return False, "El teléfono es muy corto"
    return True, "Teléfono válido"

File: visual_basic/programacion_modular/crear_modulo_de_validaciones/validaciones.vb
Module validaciones
    Public Function ValidarNombre(nombre As String) As Boolean
        If nombre.Trim() = "" Then
            Console.WriteLine("El nombre no puede estar vacío.")
            Return False
        End If
        Return True
    End Function
    Public Function ValidarTelefono(telefono As String) As Boolean
        If telefono.Trim() = "" Then
            Console.WriteLine("El teléfono no puede estar vacío.")
            Return False
        End If
        If Not IsNumeric(telefono) Then
            Console.WriteLine("El teléfono debe contener solo números.")
            Return False
        End If
        Return True
    End Function
End Module